main/dev/css/components/avatar/avatar.html

```diff
@@ -42,43 +42,81 @@ <h2 class="pad-10">components</h2>
+                            style="width: 778px; height: 204px; border:0; transform: scale(1);"
+                            sandbox="allow-scripts allow-same-origin">
+                        </iframe>
+                    </div>
                 </div>
 
-                <div class="avatarSquare">
-                    <img class="avatar-square" src="../../../asserts/avatarMale.jpg" alt="avatar">
+                <div class="items-center">
+                    <h2>Square-Avatar</h2>
+                    <div class="avatarSquare box">
+                        <!-- <img class="avatar-square" src="../../../asserts/avatarMale.jpg" alt="avatar"> -->
 
-                    <img class="avatar-square avatar-xs" src="../../../asserts/avatarMale.jpg" alt="avatar">
+                        <img class="avatar-square avatar-xs" src="../../../asserts/avatarMale.jpg" alt="avatar">
 
-                    <img class="avatar-square avatar-sm" src="../../../asserts/avatarMale.jpg" alt="avatar">
+                        <img class="avatar-square avatar-sm" src="../../../asserts/avatarMale.jpg" alt="avatar">
 
-                    <img class="avatar-square avatar-md" src="../../../asserts/avatarMale.jpg" alt="avatar">
+                        <img class="avatar-square avatar-md" src="../../../asserts/avatarMale.jpg" alt="avatar">
 
-                    <img class="avatar-square avatar-lg" src="../../../asserts/avatarMale.jpg" alt="avatar">
+                        <img class="avatar-square avatar-lg" src="../../../asserts/avatarMale.jpg" alt="avatar">
+
+                        <img class="avatar-square avatar-xl" src="../../../asserts/avatarMale.jpg" alt="avatar">
+                    </div>
+                    <div class="code">
+                        <iframe
+                            src="https://carbon.now.sh/embed?bg=rgba%2850%2C50%2C130%2C1%29&t=vscode&wt=none&l=auto&width=680&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=13px&ph=7px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520class%253D%2522avatar-square%2520avatar-xs%2522%2520src%253D%2522..%252F..%252F..%252Fasserts%252FavatarMale.jpg%2522%2520alt%253D%2522avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar-square%2520avatar-sm%2522%2520src%253D%2522..%252F..%252F..%252Fasserts%252FavatarMale.jpg%2522%2520alt%253D%2522avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar-square%2520avatar-md%2522%2520src%253D%2522..%252F..%252F..%252Fasserts%252FavatarMale.jpg%2522%2520alt%253D%2522avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar-square%2520avatar-lg%2522%2520src%253D%2522..%252F..%252F..%252Fasserts%252FavatarMale.jpg%2522%2520alt%253D%2522avatar%2522%253E%250A%253Cimg%2520class%253D%2522avatar-square%2520avatar-xl%2522%2520src%253D%2522..%252F..%252F..%252Fasserts%252FavatarMale.jpg%2522%2520alt%253D%2522avatar%2522%253E"
+                            style="width: 778px; height: 177px; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
use width100% or make a class in main css for iframe!
it will make your carbon frame responsive